specs/fulu/beacon-chain.md

```diff
@@ -151,6 +167,64 @@ class BeaconState(Container):
 
 ### Misc
 
+#### New `BlobScheduleEntry`
+
+```python
+@dataclass
+class BlobScheduleEntry(object):
+    epoch: Epoch
+    max_blobs_per_block: uint64
+```
+
+#### New `get_max_blobs_per_block`
+
+*[New in EIP7892]* This schedule defines the maximum blobs per block limit for a
+given epoch.
+
+```python
+def get_max_blobs_per_block(epoch: Epoch) -> uint64:
+    """
+    Return the maximum number of blobs that can be included in a block for a given epoch.
+    """
+    for entry in sorted(BLOB_SCHEDULE, key=lambda e: e["EPOCH"], reverse=True):
+        if epoch >= entry["EPOCH"]:
+            return entry["MAX_BLOBS_PER_BLOCK"]
+    return MAX_BLOBS_PER_BLOCK_ELECTRA
+```
+
+#### Modified `compute_fork_digest`
+
+*Note:* The `compute_fork_digest` helper is updated to account for
+Blob-Parameter-Only forks.
+
+```python
+def compute_fork_digest(
+    current_version: Version,
+    genesis_validators_root: Root,
+    current_epoch: Epoch,  # [New in Fulu:EIP7892]
+) -> ForkDigest:
+    """
+    Return the 4-byte fork digest for the ``current_version`` and ``genesis_validators_root``,
+    with a XOR bitmask of the big-endian byte representation of current max_blobs_per_block.
+
+    This is a digest primarily used for domain separation on the p2p layer.
+    4-bytes suffices for practical separation of forks/chains.
+    """
+    base_digest = compute_fork_data_root(current_version, genesis_validators_root)[:4]
+
+    if current_epoch < FULU_FORK_EPOCH:
+        return base_digest
+
+    # Find the blob parameters applicable to this epoch
+    max_blobs_per_block = get_max_blobs_per_block(current_epoch)
+
+    # Safely bitmask blob parameters into the digest
```

Comment:
Might want to mention that this is safe because `max_blobs_per_block` is limited by `MAX_BLOB_COMMITMENTS_PER_BLOCK` (4096) which is always going to fit in 4-bytes. I believe `value.to_bytes(4, "big")` will throw an exception if it doesn't fit too.

Reply:
Resolved in b00d6a179.